data-client/src/main/java/org/wikipedia/dataclient/mwapi/MwQueryPage.java

```diff
@@ -141,6 +151,27 @@ public void appendTitleFragment(@Nullable String fragment) {
         title += "#" + fragment;
     }
 
+    public boolean checkWhetherFileIsUsedInWikis() {
+        if (globalUsages == null || globalUsages.size() == 0) {
+            if (fileUsages == null || fileUsages.size() == 0) {
+                return false;
+            }
+
+            final int totalCount = fileUsages.size();
+            int ignoreCount = 0;
+
+            for (final FileUsage cur : fileUsages) {
+                if (cur.title().contains("User:Didym/Mobile upload")) {
```

Comment:
Add comment: `Ignore usage under https://commons.wikimedia.org/wiki/User:Didym/Mobile_upload/ which has been a gallery of all of our uploads since 2014.`